aws-ts-eks-migrate-nodegroups/README.md

```diff
@@ -1,8 +1,9 @@
-# examples/aws-ts-eks-migrate-nodegroups
+# Migrating EKS Node Groups with Zero Downtime
```

Comment:
Should we standardize on using imperatives ("Host a website") or gerunds ("Migrating EKS")?

Reply:
I like imperatives, but our website uses gerunds a lot. I've avoided using verbs here. Joe changed this to "Zero Downtime Migration" on the site so I'll lead with that.